src/clojure/nrepl/cmdline.clj

```diff
@@ -239,59 +244,193 @@
              options
              options))
 
+(defn args->cli-options
+  "Takes CLI args list and returns vector of parsed options map and
+  remaining args."
+  [args]
+  (let [[options _args] (split-args (expand-shorthands args))
+        merge-config (partial merge config/config)
+        options (-> options
+                    (keywordize-options)
+                    (parse-cli-values)
+                    (merge-config))]
+    [options _args]))
+
+(defn help-command
```

Comment:
I'd use names like `display-version` which are more descriptive than the very generic something-command.

Reply:
The intention was to match the CLI commands to make it easier to identify but if you think it's better to focus on idiomatic Clojure I have no objections.

Reply:
I got your idea, but I'd rather stick to idiomatic Clojure.